grid.css

```diff
@@ -0,0 +1,18 @@
+* {
```

Comment:
Bootstrap has its own grid system which is much more comprehensive than the grid systems we built together to understand the concept, please remove these and use the grid system provided by Bootstrap in this project 👍